data/featuredstory.yml

```diff
@@ -6,7 +6,7 @@ defaultRight: |
           </svg>
         </p>
         <p>Sign up to our <br>Mailing list</p>
-        <p><a class="button btn btn-primary" href="https://accounts.eclipse.org/mailing-list/openpass-wg">Subscribe</a></p>
+        <p><a style="float:none" class="button btn btn-primary" href="https://accounts.eclipse.org/mailing-list/openpass-wg">Subscribe</a></p>
```

Comment:
After spending more time on this, the issue is simply that we need to update the class in https://github.com/EclipseFdn/openpass.eclipse.org/blob/master/less/styles.less